_includes/layouts/base.html

```diff
@@ -10,12 +10,9 @@
       {%  assign was_redirected = true %}
       {% include "meta_redirect.html" %}
     {% elsif page.url == "/" %}
-      {%  assign was_redirected = true %}
-      <meta name="robots" content="noindex" />
-      <meta http-equiv="refresh" content="0;URL='https://18f.gsa.gov/guides'" />
+      {% assign was_redirected = true %}
+      {%  include "meta_redirect.html", is_index: true %}
```

Comment:
I think this works! If we want to get rid of the `elsif` (since it mostly repeats the same body as the `if`), we could try 
```
{% assign is_index = false %}
{% if page.url == "/" %}
  {% assign is_index = true %}
{% endif %}
```

Reply:
Good call, but it took a little extra. See https://github.com/18F/guides/pull/126/commits/60d781c7cf4c89dcd463d3d9538ca3de3ed05ca0